extensions/wikia/Blogs/BlogTemplate.php

```diff
@@ -258,6 +258,7 @@ class BlogTemplateClass {
 	private static $aOptions	= array( );
```

Comment:
The changes in this file are for [SOC-380](https://wikia-inc.atlassian.net/browse/SOC-380)